e2e/commons-test.mk

```diff
@@ -13,10 +13,21 @@ STACK_VERSION?=
 ELASTIC_APM_ENVIRONMENT?=local
 
 ifeq ($(ELASTIC_APM_ACTIVE),true)
+# If running in the CI then let's use the environment
+# variables from the withOtelEnv step.
+ifdef CI
```

Comment:
`CI` env variable is defined by Jenkins:

![image](https://user-images.githubusercontent.com/2871786/130797323-f224bae9-beca-4ebd-a6c8-211b6495b5da.png)